pkg/reconciler/trustedkeys/controller.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trustedkeys
+
+import (
+	"context"
+
+	"k8s.io/client-go/tools/cache"
+	kubeclient "knative.dev/pkg/client/injection/kube/client"
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/logging"
+
+	// Use the informer factory that restricts only to our namespace. This way
+	// we won't have to grant too broad RBAC rights, nor have trouble starting
+	// up if we don't have them.
+
+	pkgreconciler "knative.dev/pkg/reconciler"
+	"knative.dev/pkg/system"
+
+	"github.com/sigstore/policy-controller/pkg/apis/config"
+	trustedkeysinformer "github.com/sigstore/policy-controller/pkg/client/injection/informers/policy/v1alpha1/trustedkeys"
+	trustedkeysreconciler "github.com/sigstore/policy-controller/pkg/client/injection/reconciler/policy/v1alpha1/trustedkeys"
+	cminformer "knative.dev/pkg/injection/clients/namespacedkube/informers/core/v1/configmap"
+)
+
+// This is what the default finalizer name is, but make it explicit so we can
+// use it in tests as well.
+const finalizerName = "trustedkeyses.policy.sigstore.dev"
```

Comment:
Drop double plural, make public, and use in the other place I saw this? 🤔

Reply:
Decided to rename (and align with the other uses in proto repo) to TrustRoot. Makes more sense that way.